lib/protoflow-blocks/doc/sys/write_socket.seq.mmd

```diff
@@ -0,0 +1,15 @@
+sequenceDiagram
+    autonumber
+    participant BlockA as Another block
+    participant WriteSocket.input as WriteSocket.input port
+    participant WriteSocket as WriteSocket block
+
+    BlockA-->>WriteSocket: Connect
+
+    loop WriteSocket process
+        BlockA->>WriteSocket: Message (proto)
+        WriteSocket->>WriteSocket: Write proto message to TCP port
+    end
```

Comment:
Let's make these descriptions symmetrical to those in [`WriteStdout`](https://docs.rs/protoflow-blocks/latest/protoflow_blocks/sys/struct.WriteStdout.html):
- "Write bytes to a TCP socket"
- "Message (Bytes)"

Reply:
fixed